config/configgrpc/configgrpc_test.go

```diff
@@ -953,7 +953,7 @@ func TestDefaultUnaryInterceptorAuthFailure(t *testing.T) {
 		authCalled = true
 		return context.Background(), expectedErr
 	}
-	handler := func(ctx context.Context, req any) (any, error) {
+	handler := func(_ context.Context, _ any) (any, error) {
```

Comment:
This can be simplify even more, same everywhere where you have all params with `_`:

```suggestion
	handler := func(context.Context, any) (any, error) {
```